GraphQL/Services/ContentItemTypeBuilder.cs

```diff
@@ -0,0 +1,51 @@
+﻿using GraphQL.Types;
+using OrchardCore.ContentManagement.GraphQL.Queries.Types;
+using OrchardCore.ContentManagement.Metadata.Models;
+using System.Linq;
+
+namespace Lombiq.TrainingDemo.GraphQL.Services
+{
+    public class ContentItemTypeBuilder : IContentTypeBuilder
+    {
+        // It's a good practice to make the argument name a const because you will reuse it in the IGraphQLFilter.
+        public const string AgeFilterName = "age";
+
+        // Here you can add arguments to every Content Type (top level) field.
+        public void Build(
+            FieldType contentQuery,
+            ContentTypeDefinition contentTypeDefinition,
+            ContentItemType contentItemType)
+        {
+            // You can check the sub-fields if your argument relies on certain content parts or content fields.
+            if (contentItemType.Fields.All(field => field.Name != "person")) return;

```

Comment:
Ezt nem egészen értem.

Reply:
Így már jobb?
```
// You can check to see if the field has any specific sub-filed, if you want to rely on its features. For
// example if you only want to apply to ContentItem fields that have the "person" sub-field (i.e. those that
// have a PersonPart). This is useful if you want to expand your content part field in another module.
```

Reply:
Az első sorban is "sub-field", de amúgy OK lesz.